chaos-framework/src/main/java/io/openchaos/checker/NacosEndConsistentChecker.java

```diff
@@ -0,0 +1,211 @@
+package io.openchaos.checker;//package io.openchaos.checker;
+//
```

Comment:
It is recommended to delete the commented code

Reply:
ok, I will make it.